packages/utils-worker/package.json

```diff
@@ -0,0 +1,114 @@
+{
+  "ava": {
+    "failFast": false,
+    "files": [
+      "dist/tests/**/*.js",
+      "!dist/tests/**/fixtures/**/*.js",
+      "!dist/tests/helpers/**/*.js"
+    ],
+    "timeout": "1m"
+  },
+  "bundleSize": 2500000,
```

Comment:
😵

Reply:
Yep, unlike `extension-browser` this one measures the bundle minified, but not gzipped. It's actually smaller than `extension-browser` in an apples-to-apples comparison, though :)